services/mealie/ts-serve/ts-serve.json

```diff
@@ -0,0 +1,19 @@
+{
+    "TCP": {
+        "443": {
+            "HTTPS": true
+        }
+    },
+    "Web": {
+        "${TS_CERT_DOMAIN}:443": {
+            "Handlers": {
+                "/": {
+                    "Proxy": "http://127.0.0.1:9000"
+                }
+            }
+        }
+    },
+    "AllowFunnel": {
+        "${TS_CERT_DOMAIN}:443": true
```

Comment:
For posterity, https://github.com/tailscale/tailscale/issues/11849#issuecomment-2211623437 fixed things when I updated the ACLs in Tailscale per this comment.